frontend/src/views/setting/panel/index.vue

```diff
@@ -77,9 +77,10 @@
                                     @change="onSave('Language', form.language)"
                                     v-model="form.language"
                                 >
+                                    <el-radio v-if="globalStore.isIntl" value="en">English</el-radio>
                                     <el-radio value="zh">中文(简体)</el-radio>
                                     <el-radio value="tw">中文(繁體)</el-radio>
-                                    <el-radio value="en">English</el-radio>
+                                    <el-radio v-if="!globalStore.isIntl" value="en">English</el-radio>
                                 </el-radio-group>
                             </el-form-item>
 
```

Comment:
The provided code snippet appears to have duplication of "English" radio options. It should be cleaned up to avoid this repetition:

```diff
@@ -77,13 +77,12 @@
                                     @change="onSave('Language', form.language)"
                                     v-model="form.language"
                                 >
-                                    <el-radio value="en">English</el-radio>
+                                    <!-- Remove duplicate English radio option -->
                                 </el-radio-group>
                             </el-form-item>

```

Additionally, you may want to consider using conditional rendering within the `el-radio` components based on some logic rather than hardcoding them like this. For example, if `globalStore.isIntl` determines whether certain fields need certain visibility or not, it might make more sense to use conditional statements instead of repeating the same HTML twice with different values.

If there's additional context about what these changes will achieve (like improving accessibility by hiding one of the radios), feel free to include that information for further clarification.